File: translation_output/no_test/gpt-4/avatar/java/python-sanitized/codeforces_92_A.py
def codeforces_92_A():
    n, k = map(int, input().split())
    i = 1
    req = 1

    while k - req >= 0:
        k -= req
        i += 1
        req = i % n if i % n != 0 else n

    print(k)

# Call the function if this is the main module
if __name__ == "__main__":
    codeforces_92_A()

File: Results/DER/Translation/translation_output/w_test/gpt-3.5/avatar/python/java-sanitized/codeforces_92_A.java
import java.io.IOException;
import java.io.InputStreamReader;
import java.io.BufferedReader;

public class codeforces_92_A {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String[] input = br.readLine().split(" ");
        int n = Integer.parseInt(input[0]);
        int m = Integer.parseInt(input[1]);
        
        int i = 0;
        int ans = 0;
        while (true) {
            if (i + 1 > m) {
                ans = m;
                break;
            }
            m -= i + 1;
            i += 1;
            i %= n;
        }
        
        System.out.println(ans);
    }
}